ui/cypress/support/helpers.js

```diff
@@ -35,55 +34,84 @@ export function gen_tranform_intercepts(dataset_id, jobName, result) {
 
 // ================== End-to-End Helpers =======================================
 
-export function createModel(modelId, overrides={}) {
 
-  console.log('Creating Seed Model', modelId);
+const username = Cypress.env('DOJO_DEMO_USER');
+const password = Cypress.env('DOJO_DEMO_PASS');
+const hasAuth = Boolean(username) && Boolean(password);
+const auth = hasAuth ? {username, password} : undefined;
 
-  const testModel = genBaseModel(modelId, overrides);
+const axiosClient = axios.create({
+  baseURL: Cypress.config('baseUrl'),
+  // timeout: 1000,
+  headers: {'Content-Type': 'application/json'}
+});
 
-  modelId = testModel.id;
 
-  // The next two requests make the tests slower when iterating/developing tests:
-  cy.request('post', '/api/dojo/models', testModel)
-    .its('body').should('include', testModel.id);
+// TODO move/use seed fn
+export function createModel(modelId, variant='base', overrides={}) {
+
+  const testModel = genBaseModel(modelId, variant, overrides);
+
+  modelId = testModel.id;
 
-  return testModel;
+  return axiosClient({
```

Comment:
We're now using axios for all of this helper file fns, since it becomes confusing to go back and forth between browser Promise and Cypress promise.